pgt/src/PGT/Output/Test/Result.hs

```diff
@@ -94,19 +86,23 @@ parseEmpty :: Parser Result
 parseEmpty = Empty <$ "(0 rows)" <* Text.endOfLine
 
 parseError :: Parser Result
-parseError = Error . NonEmpty.fromList <$> Text.many1' parseErrorResult
+parseError = Error <$> parseErrorResult
   where
     parseErrorResult :: Parser ErrorResult
     parseErrorResult = do
-      error   <- parseMessage "ERROR:"
-      detail  <- parseOptionalMessage "DETAIL:"
-      context <- parseOptionalMessage "CONTEXT:"
-      line    <- optional parseLineError
+      error   <- parseMessage "ERROR:" <|> parseMessage "WARNING:"
```

Comment:
__question:__ Given `WARNING:` does not appear in the original code, should this be considered a `Refactor` commit?